internal/session/repository_session.go

```diff
@@ -263,38 +264,38 @@ func (r *Repository) ListSessions(ctx context.Context, opt ...Option) ([]*Sessio
 	case 0:
 	case 1:
 		inClauseCnt += 1
-		where, args = append(where, fmt.Sprintf("scope_id = @%d", inClauseCnt)), append(args, sql.Named(fmt.Sprintf("%d", inClauseCnt), opts.withScopeIds[0]))
+		where, args = append(where, fmt.Sprintf("scope_id = @%d", inClauseCnt)), append(args, sql.Named(strconv.Itoa(inClauseCnt), opts.withScopeIds[0]))
```

Comment:
Hm, I made these changes based on my comment, but I see this pattern is pretty common in this file, so maybe I should revert it again. Thoughts @tmessi?

Reply:
Yea I have noticed this in a lot of the repositories. I think this should be changed, but probably as a separate commit that addresses it in all places, so for now I would revert this.

Reply:
https://github.com/hashicorp/boundary/pull/2047